### PR TITLE
A rousing finale

### PR DESCRIPTION
**Pull request by gvwilson**

### CODE REVIEW

File: chapters/finale.Rmd [hunk 1]
```diff
@@ -45,17 +41,33 @@ He's added better error handling so he and his
...
+The climate crisis is the greatest threat our species has faced since civilization began.
+The COVID-19 pandemic has shown just how ill-prepared we are to deal with problems of that magnitude,
+or with the suspicion and disinformation that now poisons every public discussion online.
+
+Every hour that a researcher *doesn't* waste re-doing a data analysis by hand
+is an hour they can spend solving a new problem;
+every meeting that ends early with a clear decision
+frees up time to explain to the public what we actually know and why it matters.
+We don't expect this book to change the world,
+but we hope that knowing to to write, test, document, package, and share your work
```
Comment (cwickham) [inline chapters/finale.Rmd]:
```suggestion
but we hope that knowing how to write, test, document, package, and share your work
```

File: chapters/finale.Rmd [hunk 2]
```diff
@@ -45,17 +41,33 @@ He's added better error handling so he and his
...
+will give you a slightly better chance of doing so.
+
+We hope you have enjoyed reading what we have written;
 if so,
-we'd enjoy hearing from you.
+we would enjoy hearing from you.
```
Comment (cwickham) [inline chapters/finale.Rmd:68]: How would someone get in touch?  Should we link to something here?

**Review by cwickham — APPROVED** [review]
Much more rousing. I like it.

File: chapters/finale.Rmd [hunk 3]
```diff
@@ -45,17 +41,33 @@ He's added better error handling so he and his
...
-but they have all made us and thousands of our colleagues more productive.
-We hope you have enjoyed reading;
+healthy team dynamics---they all take time to learn,
+but they enable researchers to get more done in less time and with less pain,
+and that's why we wrote this book.
+The climate crisis is the greatest threat our species has faced since civilization began.
+The COVID-19 pandemic has shown just how ill-prepared we are to deal with problems of that magnitude,
+or with the suspicion and disinformation that now poisons every public discussion online.
+
+Every hour that a researcher *doesn't* waste re-doing a data analysis by hand
```
Comment (joelostblom) [inline chapters/finale.Rmd]: Maybe also add an example that directly mentions something we taught e.g. "...or struggling to use poorly documented software packages"?

**Review by joelostblom — APPROVED** [review]
Looks geat to me!

File: chapters/finale.Rmd (hunk 1 above)
Comment (gvwilson) [reply chapters/finale.Rmd]: :+1: thanks for catching this

File: chapters/finale.Rmd (hunk 2 above)
Comment (gvwilson) [reply chapters/finale.Rmd:68]: I'll add links - thanks.

File: chapters/finale.Rmd (hunk 3 above)
Comment (gvwilson) [reply chapters/finale.Rmd]: changed it to "wrestling with software" - shorter and more general.

File: chapters/finale.Rmd [hunk 4]
```diff
@@ -56,18 +56,24 @@ The climate crisis is the greatest threat our species has faced since civilizati
...
 We don't expect this book to change the world,
-but we hope that knowing to to write, test, document, package, and share your work
+but we hope that knowing how to write, test, document, package, and share your work
 will give you a slightly better chance of doing so.
-
 We hope you have enjoyed reading what we have written;
 if so,
 we would enjoy hearing from you.
 
+-   Damien Irving (<https://drclimate.wordpress.com/>)
```
Comment (DamienIrving) [inline chapters/finale.Rmd]:
```suggestion
-   Damien Irving (<https://damienirving.github.io/>)
```

**Pull request merged**